tests/python/test_multidevice.py

```diff
@@ -513,8 +513,9 @@ def multidevice_schedule(self) -> None:
                 self.k_grad,
                 self.v_grad,
             ]
+            non_sharded_tvs = [self.seed, self.offset]
 
-            for t in input_tvs + output_tvs:
+            for t in input_tvs + output_tvs + non_sharded_tvs:
```

Comment:
```suggestion
            for t in [self.q, self.k, ...]:  # list all TVs here
```

I forgot why we tried to distinguish inputs and outputs in this particular test. They all seem to be treated the same way.

Reply:
For `output_tvs` I was demonstrating use of TransformPropagator through `transform_like` and hence separated them.